podium-gateway/src/main/java/nl/thehyve/podium/service/RequestReviewProcessService.java

```diff
@@ -161,23 +161,19 @@ private void setVariable(RequestReviewProcess requestReviewProcess, ReviewVariab
             value);
     }
 
-    private static ActionNotAllowedInStatus actionNotAllowedInStatus(RequestReviewStatus status) throws ActionNotAllowedInStatus {
-        return new ActionNotAllowedInStatus("Invalid action in status: " + status.name());
-    }
-
     public RequestReviewProcess submitForValidation(AuthenticatedUser user, RequestReviewProcess requestReviewProcess) throws ActionNotAllowedInStatus {
         if (requestReviewProcess.getStatus() == RequestReviewStatus.Revision) {
             return completeCurrentTask(user, requestReviewProcess);
         }
-        throw actionNotAllowedInStatus(requestReviewProcess.getStatus());
+        throw ActionNotAllowedInStatus.forStatus(requestReviewProcess.getStatus());
```

Comment:
Renaming the exception to ActionNotAllowed and then having the method .forStatus makes it nicer to read. I believe we fairly consistently use the .forStatus method already throughout the application.